test/test_test_circuit_crx_gate.py

```diff
@@ -18,13 +18,23 @@ class TestTestCircuitCRXGate(unittest.TestCase):
     $
     """
 
+    def _u3(self, parameter: list) -> np.ndarray:
+        theta, phi, lambda_ = parameter
+        return np.array([
+            [np.cos(theta/2), -np.exp(1j*lambda_) * np.sin(theta/2)],
+            [np.exp(1j*phi)*np.sin(theta/2),
+             np.exp(1j*(lambda_ + phi))*np.cos(theta/2)]])
+
+    def _rx(self, parameter: list) -> np.ndarray:
+        theta = parameter[0]
+        return self._u3([theta, -np.pi/2, np.pi/2])
+
     def test_crx_regular_qubit_order(self,):
         circ = TestCircuit(2)
         lambda_ = np.pi/8
-        actual_gate = circ._create_all_qubit_gate_from_crx_gate(
-            control_qubit=[0], target_qubit=[1], control_value=[1],
-            parameter=[lambda_]
-        )
+        actual_gate = circ._create_all_qubit_gate_from_original_qubit_gate(
+            self._rx([lambda_]),
```

Comment:
I think that not  `self._rx([lambda_])` but `circ._rx([lambda_])` should be used here, because the latter is in actual used in the source code.

Reply:
Modified! Please review it again! Thanks in advance!

Reply:
Thx!